### PR TITLE
Fixed Cloud-Config examples

### PR DESCRIPTION
**Pull request by kayrus**
/cc @robszumski

### CODE REVIEW

File: Documentation/cloud-config.md [hunk 1]
```diff
@@ -244,7 +244,7 @@ Each item is an object with the following fields:
 
 **NOTE:** The command field is ignored for all network, netdev, and link units. The systemd-networkd.service unit will be restarted in their place.
 
-##### Examples
+#### Examples
```
Comment (crawford) [inline Documentation/cloud-config.md]: Does it make sense to move this up one more level so it's directly under Configuration Parameters?

**Comment by crawford** [comment]
etcd stuff looks good

File: Documentation/cloud-config.md (hunk 1 above)
Comment (kayrus) [inline Documentation/cloud-config.md]: Reverted back. There is an issue in online doc processor. For example: https://coreos.com/os/docs/latest/cloud-config.html#examples

**Comment by kayrus** [comment]
Reverted "Examples" back. There is an issue in coreos.com/docs engine. For example: https://coreos.com/os/docs/latest/cloud-config.html#examples "Examples are not bold"

**Comment by crawford** [comment]
@kayrus Sorry, I was suggesting `### Examples`. "up" is a little ambiguous.

**Comment by robszumski** [comment]
@kayrus what is the issue? I don't follow what you both are talking about.

**Comment by kayrus** [comment]
@robszumski "Examples" text is not bold. We've talked about that two weeks ago.

**Comment by robszumski** [comment]
I'm not seeing how the the cloud-config locations doc would help you out. It doesn't allow you to easily map those locations to platforms, so you're just supposed to check all locations?

While you may be able to find the config on disk, we need to mention that you have to update it elsewhere (via openstack API etc) since we're pulling it down.

**Idea**: is it better to keep mentioning these in each provider's docs so we can tell them exactly how to update the config, where it is, etc?

**Comment by robszumski** [comment]
Sidenote: We should mention `journalctl _EXE=/usr/bin/coreos-cloudinit` somewhere in the cloud-config doc.

**Comment by kayrus** [comment]
Done

**Comment by kayrus** [comment]
Added cloud-config location for Azure

File: Documentation/cloud-config.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ CoreOS allows you to declaratively customize various OS-level items, such as net
 
 Your cloud-config is processed during each boot. Invalid cloud-config won't be processed but will be logged in the journal. You can validate your cloud-config with the [CoreOS validator]({{site.url}}/validate) or by running `coreos-cloudinit -validate`.
 
+In addition to `coreos-cloudinit -validate` command you can debug `coreos-cloudinit` system output through the `journalctl` tool:
```
Comment (crawford) [inline Documentation/cloud-config.md]: Can you also mention coreos.com/validate?

**Comment by kayrus** [comment]
Have mentioned.

File: Documentation/cloud-config-locations.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Cloud-Config Locations
+
+Every CoreOS boot Cloud-Config tool looks for a config file to configure your host. Here is a list of locations which are used by the Cloud-Config utility, depending on your CoreOS platform:
```
Comment (crawford) [inline Documentation/cloud-config-locations.md]: "On every boot, coreos-cloudinit looks..."

File: Documentation/cloud-config-locations.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+* Mount point with [config-2]({{site.baseurl}}/os/docs/latest/config-drive.html#contents-and-format) label. It should contain a `openstack/latest/user_data` relative path. An absolute path inside CoreOS should look like `/media/configvirtfs/openstack/latest/user_data`. Usually used by cloud providers or in VM installations.
```
Comment (crawford) [inline Documentation/cloud-config-locations.md]: This link is not valid. Don't worry about "{{site.baseurl}}", there is a script which will rewrite the link.

**Comment by kayrus** [comment]
Fixed

**Comment by kayrus** [comment]
Any news?

**Pull request merged**